Makefile

```diff
@@ -722,8 +722,16 @@ package-repo-bundle: tools prep-package-tools ## Build tar bundles for package r
 	cd hack/packages/package-tools && $(GO) run main.go repo-bundle generate --repository=$(PACKAGE_REPOSITORY) --registry=$(OCI_REGISTRY)/packages/$(PACKAGE_REPOSITORY) --version=$(REPO_BUNDLE_VERSION) --package-values-file=$(PACKAGE_VALUES_FILE) --sub-version=$(REPO_BUNDLE_SUB_VERSION)
 
 .PHONY: push-package-bundles
-push-package-bundles: tools  prep-package-tools ## Push package bundles
-	cd hack/packages/package-tools && $(GO) run main.go package-bundle push --repository=$(PACKAGE_REPOSITORY) --registry=$(OCI_REGISTRY)/packages/$(PACKAGE_REPOSITORY) --version=$(PACKAGE_VERSION) --sub-version=$(PACKAGE_SUB_VERSION)
+ifeq (push-package-bundles, $(firstword $(MAKECMDGOALS))) ## Allow passing in arguments to the make command
+  push_package_bundles_args := $(wordlist 2, $(words $(MAKECMDGOALS)), $(MAKECMDGOALS))
+  $(eval $(push_package_bundles_args):;@:)
+endif
+push-package-bundles: tools prep-package-tools ## Push specified package bundle(s) in a package repository
```

Comment:
can we just pass another variable to the make target to make this easy? `PACKAGE_BUNDLES` would be a list of packages delimited by a `,`?
ex: `PACKAGE_REPOSITORY=management OCI_REGISTRY=docker.io/foo PACKAGE_BUNDLES=featuregates,core-management-plugins make push-package-bundles `

and in the command's run function, you can split the string and use the package bundle names to push them to the remote registry? I think this would be much cleaner and we can get rid of [lines](https://github.com/yharish991/tanzu-framework/pull/2/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R725-R728)